.github/workflows/check_notebooks.yml

```diff
@@ -0,0 +1,35 @@
+name: Jupyter Notebooks CI
+on: [push, pull_request]
+jobs:
+  Check-Notebooks:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -el {0}
+    steps:
+      - name: check out repository code
+        uses: actions/checkout@v3
+      - name: set up conda environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          auto-activate-base: false
+          activate-environment: notebook-env
+          python-version: 3.10.6
```

Comment:
I certainly agree with you that this would be a more flexible solution. However, as far as I know, the `setup-miniconda` action can only install either the _latest_ or a _specific_ python version. In principle we could also switch to the latest python version.

Reply:
I'm inclined to use the latest python version as that's what we do in the docker CI.

Reply:
Thanks for the answers! I think consistency is best, so should be change to latest to match the docker CI?